token/cli/src/main.rs

```diff
@@ -1611,8 +1611,8 @@ fn main() {
             path: default_signer_path,
             arg_name: default_signer_arg_name,
         };
-        // Owner doesn't sign when it's a multisig
-        let owner = if matches.is_present(MULTISIG_SIGNER_ARG.name) {
+        // Owner doesn't sign when it's a multisig; and the `accounts` command is read-only
+        let owner = if matches.is_present(MULTISIG_SIGNER_ARG.name) || sub_command == "accounts" {
```

Comment:
I was kinda wondering why owner doesn't get mapped to a `NullSigner`

Reply:
https://github.com/solana-labs/solana/blob/e1ceb430e3bc51925a3d765677691c4fe9b4187f/clap-utils/src/keypair.rs#L196

Reply:
Yeah I was just pondering that block.  I'm wondering if it'd make sense to always stuff pubkeys into a `NullSigner` then wait and fail signature checks...  a problem for another PR in any case...

Back on topic!

I think this is fine as is.  It's a pity that `accounts` is the only RO subcommand that doesn't take its target positionally, but :shrug:

Reply:
I rather like that idea of depending on signature checks to fail NullSigner.

Yeah, the positional mint arg irks me, actually. I suppose we could break `accounts` to take an explicit `--mint` arg, and use the positional for the owner instead...
I'm going to merge this as-is, but let's ponder that.

Reply:
Changing the meaning of the positional arg and using an explicit `--mint` arg would also make this issue go away :)
https://github.com/solana-labs/solana-program-library/issues/762